src/components/PathwaysList/PathwaysList.tsx

```diff
@@ -94,9 +94,9 @@ const PathwaysList: FC<PathwaysListProps> = ({ evaluatedPathways, callback, serv
       </Button>
 
       <PathwaysTable
-        pathways={evaluatedPathways}
+        pathways={pathways}
         deleteButton={(): void => callback()}
```

Comment:
Sorry I forgot lint doesn't want it to be empty
```suggestion
        deleteButton={(): void => {
             // do nothing
        }}
```